st2actions/st2actions/scheduler/handler.py

```diff
@@ -62,6 +62,9 @@ def __init__(self):
         self.message_type = LiveActionDB
         self._shutdown = False
         self._pool = eventlet.GreenPool(size=cfg.CONF.scheduler.pool_size)
+        self._execution_scheduling_timeout_threshold_ms = \
+                cfg.CONF.scheduler.execution_scheduling_timeout_threshold_m * \
+                EXECUTION_SCHEDUELING_TIMEOUT_THRESHOLD_MIN
```

Comment:
Please rename the private var to `self._scheduling_timeout_threshold_ms`. The math should be just `self. _scheduling_timeout_threshold_ms  = cfg.CONF.scheduler.scheduling_timeout_threshold_min * 60 * 1000`.  We don't need a constant for min -> ms conversion.  Also, please move the comments above at the EXECUTION_SCHEDUELING_TIMEOUT_THRESHOLD_MS constant to here.